README.md

```diff
@@ -45,6 +45,12 @@ Before running the app, please install Docker first
 
 ### Backend
 
+- Create a `server/src/.env` (in `server/src`, not in the project home dir) file and set all required variables
```

Comment:
Only this is required. Everything else should be removed

Reply:
Why? Docker Compose can't access the `.env` file without it being explicitly specified in the `docker-compose.yml` file. So how come that `docker-compose.yml` doesn't need to be modified?

Reply:
The .env file will be in the [docker](https://github.com/apssouza22/chatflow/blob/main/docker-compose.yml#L18) and the Python it self will read the .env file

Reply:
In fact the variables in the docker-compose can be removed

Reply:
@apssouza22 Wait, following the README.md, I run `server.py` plainly, not in Docker. But, apparently, Python still can (not looked into the source) read the `.env`, because it is in the same directory as the Python script.

Reply:
You can run the project using docker and without docker. 

The project read the configuration from env variables or from the .env file.

The .env file is available outside Docker and inside docker. The parameters in the .env is to run without Docker (db configs, etc). When inside docker we need to override the parameters through env variables because the addresses changes.

The Docker compose have those variables set with fallback. So it doesn't need any .env. 

Everything is correct except the path in the readme

Reply:
I have fixed the readme. So I am closing this MR